lua/lazyvim/plugins/extras/ai/claudecode.lua

```diff
@@ -0,0 +1,22 @@
+return {
+  "coder/claudecode.nvim",
+  opts = {},
+  keys = {
+    { "<leader>a", "", desc = "+ai", mode = { "n", "v" } },
+    { "<leader>ac", "<cmd>ClaudeCode<cr>", desc = "Toggle Claude" },
+    { "<leader>af", "<cmd>ClaudeCodeFocus<cr>", desc = "Focus Claude" },
+    { "<leader>ar", "<cmd>ClaudeCode --resume<cr>", desc = "Resume Claude" },
+    { "<leader>aC", "<cmd>ClaudeCode --continue<cr>", desc = "Continue Claude" },
+    { "<leader>ab", "<cmd>ClaudeCodeAdd %<cr>", desc = "Add current buffer" },
+    { "<leader>as", "<cmd>ClaudeCodeSend<cr>", mode = "v", desc = "Send to Claude" },
+    {
+      "<leader>as",
+      "<cmd>ClaudeCodeTreeAdd<cr>",
+      desc = "Add file",
+      ft = { "NvimTree", "neo-tree", "oil" },
```

Comment:
`mini.files` should be included as well

```suggestion
      ft = { "NvimTree", "neo-tree", "oil", "minifiles" },
```